detection-rules/deceptive_pdf_attachment_mention.yml

```diff
@@ -0,0 +1,44 @@
+name: "Self-sent fake PDF attachment with misleading link"
+description: "Detects messages sent from a user to themselves containing a fake PDF icon from Google's CDN, claiming to have an attachment while only containing images, and including links to suspicious subdomains that appear to be PDF files."
+type: "rule"
+severity: "low"
+source: |
+  type.inbound
+  // pdf image
+  and strings.contains(body.html.raw,
+                       'https://ci3.googleusercontent.com/meips/ADKq_Naq6rm1GwC4XYZepCUQtEMnJ-r-HjyX_C5lBU7lpxQk1OIDV7vvQYvSJQWYmQCzG8moTgX3Wak625OtyHWRinVeUJs7K710JiIZ4JNXVpTmC8PJjV4K34GsBA=s0-d-e1-ft#https://res-1.cdn.office.net/assets/mail/file-icon/png/pdf_16x16.png'
+  )
+  // mentions attachments but there are none or just images with no pdfs
+  and strings.starts_with(body.current_thread.text, 'Please see attached.')
+  and all(attachments, .file_type in $file_types_images)
+  //self sender
+  and (
+    length(recipients.to) == 1
+    and sender.email.email == recipients.to[0].email.email
+  )
+  // display text ends with .pdf and link is a free subdomain host
+  and any(body.current_thread.links,
+          (
+            .href_url.domain.root_domain in $free_subdomain_hosts
+            or strings.parse_domain(.href_url.query_params_decoded["domain"][0]).root_domain in $free_subdomain_hosts
```

Comment:
reference something more like this, it might need tweaked into this usecase. 

```
             // Mimecast link logic
             or (
               .href_url.domain.root_domain in (
                 "mimecastprotect.com",
                 "mimecast.com"
               )
               and any(.href_url.query_params_decoded['domain'],
                       strings.parse_domain(.).root_domain in $free_subdomain_hosts
               )
```